Lib9c/Action/ClaimItems.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Extensions;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [ActionType(ActionTypeText)]
+    public class ClaimItems : GameAction, IClaimItems
+    {
+        private const string ActionTypeText = "claim_items";
+
+        public List<(Address address, List<FungibleAssetValue> fungibleAssetValues)> ClaimData { get; private set; }
```

Comment:
In my thought, I want to suggest using `IReadOnlyList<T>` because it can be corrupted. (like JavaScript's `const` does.)

```javascript
const list = [];
list.push(111); // list != []
```

```csharp
var claimItems = ClaimItems();
claimItems.ClaimData.Add((new Address("..."), new List<FungibleAssetValue> { ... }));  // 🙀 🙀 🙀 🙀 🙀 🙀 
```